packages/xod-client-electron/src/upload/messages.js

```diff
@@ -1,5 +1,7 @@
 export default {
-  SUCCESS: {
-    title: 'Uploaded successfully',
-  },
+  CANT_COMPILE_IN_CLOUD_FOR_BOARD: ({ boardName }) => ({
+    title: `Can not compile the code in the cloud`,
```

Comment:
Does not follows the headlinesse style. Can be more informative:

Cloud compilation not supported